ls1intum/Artemis
# [Feature] Show Lecture Unit Release Date and Version to Instructor

### Checklist

### Motivation and Context


@krusche requested that instructors can see the release date and attachment version of lecture units.

### Description

Added badges for the release date and attachment version

### Steps for Testing

Open a lecture unit management page and check that the new badges appear (see screenshot)

### Test Coverage



Simple change, no change

### Screenshots


![image](https://user-images.githubusercontent.com/29718932/115147287-9c04e780-a05a-11eb-9543-5bff3374795e.png)
